carma_novatel_driver_wrapper/launch/carma-novatel-driver-wrapper-launch.py

```diff
@@ -0,0 +1,59 @@
+
+#  Copyright (C) 2021 LEIDOS.
+
+#  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+#  use this file except in compliance with the License. You may obtain a copy of
+#  the License at
+
+#  http://www.apache.org/licenses/LICENSE-2.0
+
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations under
+#  the License.
+
+
+from ament_index_python.packages import get_package_share_directory
+
+from launch import LaunchDescription
+from launch_ros.actions import Node
+
+from launch.actions import DeclareLaunchArgument
+from launch.substitutions import LaunchConfiguration
+
+from launch.actions import IncludeLaunchDescription
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+
+def generate_launch_description():
+    
+    log_level = LaunchConfiguration('log_level')
+    declare_log_level_arg = DeclareLaunchArgument(
+        name ='log_level', default_value = 'WARN')
+    
+    carma_novatel_driver_wrapper = Node(
+        package = 'carma_novatel_driver_wrapper',
+        executable = 'carma_novatel_wrapper_node',
+        name='carma_novatel_driver_wrapper',
+        arguments=['--ros-args', '--log-level', log_level ]
```

Comment:
The parameter file has not been loaded. 
Also I just realized that because the parameter file is in the component format it won't work with the regular node tag. It would probably be best if you loaded this as a component following the format used in the template_package.